conans/client/tools/files.py

```diff
@@ -11,6 +11,39 @@
 from conans.errors import ConanException
 from conans.util.files import load, save, _generic_algorithm_sum
 
+from psutil import NoSuchProcess
```

Comment:
We tried to use psutil to implemented PID locks, and we had to remove it, because psutil fails to install properly in different platforms. If it is a transitive dependency of gnupg, then something has to be abstracted or improved in order to use it.

Reply:
In fact it seems that psutil is optional as far as the code in python-gnupg goes. 
It is only used in one module gnupg._meta, and then only to detect gpg-agent, which this PR doesn't rely on at all. For that reason it is ok to run without psutil (this is specifically done in python-gnupg so it can run in PyPy) . However, it is in the requirements.txt file, and I am not enough of an expert to know how we can force it to not be installed....

As far as this PR goes, the psutil is only there to catch that exception, which only seems to occur on python 2.7 on Windows, so in fact I believe this could be replaced with a "catch all" exception handler, and somehoe force not to install psutil and it would still work.

Reply:
The problem is that ``pip install conan`` miserably fails because of failure to install psutil. So no matter what you do in the python code, it is already broken at install.

Reply:
yes, there seems no way round it. Fortunately, it looks like one of the other forks of python-gnupg (there are 3 at least) which is also on PyPI doesn't require psutil and is very similar. I will attempt to port this PR to that fork

Reply:
Ok, great, let me know how it goes. 
If not, we will try to push the "plugins" feature (at least start considering it for 1.1). Thanks!